src/Tables/atoms/TableRowThumbnail.tsx

```diff
@@ -84,65 +86,51 @@ interface IProps {
   aspect?: VideoAspects
   mediaUrl?: string
   mediaType?: IMediaType
+  retryImageLoad?: boolean
+  retryLimit?: number;
 }
 
 // Image
 // No Image Placeholder
 
-const TableRowThumbnail: React.FC<IProps> = ({ hoverZoom = true, image, mediaUrl, mediaType }) => {
-
-  const [hasPreview, setHasPreview] = useState(false);
-  const [isImageValid, setIsImageValid] = useState(false);
-  const { createMediaModal, isMediaUrlValid } = useMediaModal();
+const TableRowThumbnail: React.FC<IProps> = ({ hoverZoom = true, image='', mediaUrl, mediaType, retryImageLoad= false, retryLimit=5}) => {
+  const [showImage, setShowImage] = useState(true);
+  const [imgSrc, setImgSrc] = useState(image);
+  const { createMediaModal } = useMediaModal();
+  const [retryCount, setRetryCount] = useState(0);
+  const imgRef = useRef<HTMLImageElement>(null);
 
   const handleModal = useCallback(async () => {
-
-    if (mediaUrl && mediaType && hasPreview) {
+    if (mediaUrl && mediaType ) {
       createMediaModal({ src: mediaUrl, mediaType });
     }
+  }, [createMediaModal, mediaType, mediaUrl]);
 
-  }, [createMediaModal, mediaType, mediaUrl, hasPreview]);
-
-  const verifyPreview = useCallback(async (currentMediaUrl: string, currentMediaType: IMediaType) => {
-    const isValidUrl: boolean = await isMediaUrlValid(currentMediaUrl, currentMediaType);
-
-    setHasPreview((prev) => {
-
-      if (prev === isValidUrl) {
-        return prev;
-      }
-      return isValidUrl;
-    });
-
-  }, [isMediaUrlValid]);
+  useEffect(()=>{
+    setShowImage(false);
+    setRetryCount(0);
+    setImgSrc(image);
+  },[image]);
 
-  useEffect(() => {
-    if ( mediaUrl && mediaType) {
-      verifyPreview(mediaUrl, mediaType);
+  useEffect(()=>{
+    if(imgRef.current && imgRef.current.complete){
+      setShowImage(true);
     }
```

Comment:
This allows for a bug. When src ="" or is cached error this allows for broken images. I will attempt a better solution.